spring-mvc/src/main/java/com/kodemon/springmvc/controllers/FightController.java

```diff
@@ -190,7 +190,12 @@ public String fightWild(@RequestParam String mode, ServletRequest r, Model model
         session.setAttribute("authenticatedUser", user);
 
         LOG.debug(user.getUserName() + (mode_ == WildPokemonFightMode.CATCH ? " catching" : " fighting") + " wild " + wildPokemon.getName().getName() + " level " + wildPokemon.getLevel() + " -> " + (fightResult ? "Success!" : "Failed"));
-        return "home";
+        wildPokemon = pokemonFacade.generateWildPokemon(user);
+        session.setAttribute("wildPokemon", wildPokemon);
+        model.addAttribute("wildPokemon", wildPokemon);
+        model.addAttribute("trainersPokemon", user.getPokemons().get(0));
```

Comment:
Isn't it covered a few lines earlier in the method, does it need to be done again?